lib/mutant/mutator/node/kwargs.rb

```diff
@@ -28,15 +28,15 @@ def emit_argument_presence
         def emit_argument_mutations
           children.each_with_index do |child, index|
             Mutator.mutate(child).each do |mutant|
-              if forbid_argument?(mutant)
+              unless forbid_argument?(mutant)
```

Comment:
I actually originally intend to negate in the name and call side, but only carried out the name.